### ISSUE TITLE
ProgressBarAndroid Component does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [react-native-progress-view/progress-bar-android#62]
facebook/react-native#30941 

> When using a screen reader the ProgressBarAndroid component does not announce that it is disabled

Any help in identifying if this issue has been addressed in this repo would be greatly appreciated!
